smalld/smalld.py

```diff
@@ -214,7 +214,7 @@ def send_request(self, method, path, payload="", attachments=None, params=None):
 
         try:
             content = res.json() if res.status_code != 204 else {}
-        except requests.RequestException:
+        except json.JSONDecodeError:
             raise HttpError(response=res)
```

Comment:
We should include something in here that says it was a json error. Probably as the cause of the HttpError?

Reply:
since this is raised from an exception handler, it's already associated (`__cause__`) with the `JSONDecodeError`. Unless we want a flag in the `HttpError` itself, or maybe a subclass.

Reply:
Nope, cause is enough